### ISSUE TITLE
Does not work in OS X 10.12 Sierra 

### ISSUE THREAD

**Issue by msdocs** [sindresorhus/macos-wallpaper#15]
I was looking for an easy way to change the wallpaper via terminal and this seemed like a solution  until it did not work. At first I was using AppleScript which works but only in the AppleScript editor 

so far the best solution I found was this 

`function wallpaper() {
    sqlite3 ~/Library/Application\ Support/Dock/desktoppicture.db "update data set value = '$1'" && killall Dock 
}`[Here](https://apple.stackexchange.com/questions/40644/how-do-i-change-desktop-background-with-a-terminal-command/145174#145174)

**Comment by sindresorhus** [comment]
"Does not work" is not very actionable. Please provide more information. I just tested and it works fine for me on Sierra.

**Comment by msdocs** [comment]
The wallpaper initially doesn't change for me when I run the script . I tested it  and image would not change. Then I opened wallpaper settings and I would run the script and it would change momentarily in settings. Today testing it again after seeing your response and it seems to be working. It seems a system bug might have been involved i need to go back and check the logs.   

Thank You for the speedy response I did not expect a response this quick .
